vulture_os/toolkit/api_parser/gatewatcher_alerts/gatewatcher_alerts.py

```diff
@@ -111,7 +111,14 @@ def get_logs(self, since, to):
             for alert in alerts.get("results", []):
                 raw_alert_url = f"https://{self.gatewatcher_alerts_host}{self.RAW_ALERTS_ENDPOINT}/{alert['uuid']}"
                 raw_alert = self.execute_query(raw_alert_url)
-                results.append({'alert': alert, 'raw_alert': raw_alert})
+
+                # Merge `raw_alert` into `alert`
+                fields_to_delete = ["id", "severity"]  # Remove duplicate fields to avoid loss of information
+                for field in fields_to_delete:
+                    del raw_alert[field]
```

Comment:
Don't risk a KeyError
```suggestion
                for field in fields_to_delete:
                    if field in raw_alert:
                        del raw_alert[field]
```